### PR TITLE
ci: Test with the required minimum docker-compose

### PR DESCRIPTION
**Pull request by BYK**
Refs #1061, refs #1062, refs #1063.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ on:
       - "release/**"
   pull_request:
 env:
-  DOCKER_COMPOSE_VERSION: 1.29.2
+  DOCKER_COMPOSE_VERSION: 1.28.0
```
Comment (chadwhitacre) [inline .github/workflows/test.yml:12]: How about a comment here and in `check-minimum-requirements.sh` that we want to keep these two values in sync?

https://github.com/getsentry/onpremise/blob/47c4a1df299fffb8340160d9633c05a6c8f75adc/install/check-minimum-requirements.sh#L4

**Review by chadwhitacre — CHANGES_REQUESTED** [review]
Comments plz to avoid confusion in the future.

File: .github/workflows/test.yml (hunk 1 above)
Comment (chadwhitacre) [reply .github/workflows/test.yml:12]: Sure, I'm fine to DRY up if you want.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -29,15 +27,16 @@ jobs:
...
-          sudo mv docker-compose /usr/local/bin
-
       - name: Checkout
         uses: actions/checkout@v2
+  
+      - name: Pin docker-compose
+        run: |
+          COMPOSE_PATH=/usr/local/bin/docker-compose
+          source ./install/_min-requirements.sh
+          sudo rm $COMPOSE_PATH
```
Comment (chadwhitacre) [inline .github/workflows/test.yml:37]: Why is the `rm` necessary if we sudo the curl call?

Comment (chadwhitacre) [reply .github/workflows/test.yml:37]: I guess `curl` itself will complain. Could use a redirect but I guess this is fine, too.

**Review by chadwhitacre — APPROVED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (BYK) [reply .github/workflows/test.yml:37]: 🤷🏻‍♂️

**Pull request merged**